CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ set(action_files
 rosidl_generate_interfaces(${PROJECT_NAME}
   ${msg_files}
   ${srv_files}
+  ${action_files}
```

Comment:
These actions won't compile on ROS2 Crystal patch 2

https://github.com/ros2/rosidl/issues/348

Reply:
I just saw that. I commented it out with a TODO